submissions/Oleksii_Drohachov/OOP-exercise/index.js

```diff
@@ -0,0 +1,73 @@
+class Creature {
+  constructor(species, name, gender, saying, friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  output() {
+    const { name, species, gender, saying, friends } = this;
+
+    return `${name}; ${species}; ${gender}; ${saying}; ${friends.join(", ")}`;
```

Comment:
There is still repetition (`"; "`). Pack whatever you need into an array and `Array#join`.
You will benefit a lot from recognizing use cases for `Array` methods.